tests/TestCase.php

```diff
@@ -7,4 +7,6 @@
 abstract class TestCase extends BaseTestCase
 {
     use CreatesApplication;
+
+    protected $baseUrl = "http://arcv-service.test";
```

Comment:
Pull from env

Reply:
ooh. can't - It wont accept expressions.